.github/scripts/test.sh

```diff
@@ -1,2 +1,3 @@
-flake8 --exit-zero --statistics --config build_tools/flake8.cfg RefRed
-pytest --cov=RefRed test/
\ No newline at end of file
+flake8 --exit-zero --statistics RefRed
+# allow tests to pass until they are all fixed
+pytest --cov=RefRed --cov-report=xml --cov-report=term test || true
```

Comment:
There are two failing tests that need to be fixed so for now I just have them succeed with `|| true`.

Reply:
looks good!

Reply:
I just fixed the last two tests and remove the `|| true`.